wrapspawner/wrapspawner.py

```diff
@@ -212,6 +213,14 @@ class ProfilesSpawner(WrapSpawner):
             the first item starts selected."""
         )
 
+    def __init__(self, *args, **kwargs):
+        super(ProfilesSpawner, self).__init__(*args, **kwargs)
+
+        keys = [p[1] for p in self.profiles]
+        if len(set(keys)) != len(keys):
+            logging.warning(
```

Comment:
If keys are not unique, it's always a good idea to say _what_ keys are repeating. That makes the message much more actionable.

Reply:
I'll do the change :+1:, thanks